MANIFEST.in

```diff
@@ -1,8 +1,11 @@
-include requirements/*.txt
-recursive-include apps/openassessment/templates *.html
-recursive-include apps/openassessment/xblock/static *
-recursive-include apps/openassessment/locale *.po
-recursive-include apps/openassessment/locale *.mo
 include LICENSE
 include AUTHORS
 include README.rst
+include openassessment/xblock/static/css/*.css
+include openassessment/xblock/static/js/openassessment*.min.js
+recursive-include openassessment/xblock/static/js/src *.js
```

Comment:
Ah cool, that makes sense.

I could see someone installing and wanting to run DEBUG mode to trace down a js error, so this seems reasonable.